src/views/Signup.js

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from "react";
+import { commitMutation } from 'react-relay'
+import environment from '../config/relay'
+import { Redirect } from "react-router";
+import graphql from 'babel-plugin-relay/macro'
+
+import {Form, Button, Container} from "react-bootstrap";
+
+const mutation = graphql`
+    mutation SignUpMutation($input: UserCreateInput!){
+        userCreate(input: $input){
+            user {
+                id
+            }
+        }
+    }
+`
+
+export default function SignUp() {
+    const [username, setUsername] = useState("");
+    const [email, setEmail] = useState("");
+    const [password, setPassword] = useState("");
+    const [redirect, setRedirect] = useState(false)
+
+  function validateForm() {
+    return email.length > 0 && password.length > 0;
+  }
+
+  function handleSubmit(e) {
+    e.preventDefault();
```

Comment:
Semicolors `;` should be left out when using [Standard JS](https://standardjs.com/) syntax. (We use babel compilers to ensure compiled JS works correctly.)